File: Livro - Módulo Java/3 - Estrutura de Decisão/Exercicio.02.java
import java.util.Scanner;

public class Exercicio2 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        // O total de R$ 780.000,00 será dividido entre os três primeiros colocados de um concurso,
        // em partes diretamente proporcionais aos pontos conseguidos por eles.
        // Construa um programa que leia o número de pontos dos três primeiros colocados e
        // imprima o valor recebido por cada um deles.

        System.out.print("Digite a quantidade de pontos do 1° colocado: ");
        double ponto1 = scanner.nextDouble();

        System.out.print("Digite a quantidade de pontos do 2° colocado: ");
        double ponto2 = scanner.nextDouble();

        System.out.print("Digite a quantidade de pontos do 3° colocado: ");
        double ponto3 = scanner.nextDouble();

        double total = ponto1 + ponto2 + ponto3;
        double valor1 = (ponto1 / total) * 780000;
        double valor2 = (ponto2 / total) * 780000;
        double valor3 = (ponto3 / total) * 780000;

        System.out.println("Primeiro colocado: R$" + valor1);
        System.out.println("Segundo colocado: R$" + valor2);
        System.out.println("Terceiro colocado: R$" + valor3);
    }
}

File: Livro - Módulo Python/3 - Estrutura de Decisão/Exercicio.02.py
#2. O total de R$ 780.000,00 será dividida entre os três primeiros colocados de um
# concurso, em partes diretamente proporcional aos pontos conseguidos por eles. 
# Construa um programa que leia o numero de pontos dos três primeiros colocados e
# imprima a importância que caberá a cada um deles.

pontos1 = float(input("Digite os pontos do primeiro colocado: "))
pontos2 = float(input("Digite os pontos do segundo colocado: "))
pontos3 = float(input("Digite os pontos do terceiro colocado: "))
total = pontos1 + pontos2 + pontos3
valor1 = (pontos1 / total) * 780000
valor2 = (pontos2 / total) * 780000
valor3 = (pontos3 / total) * 780000
print("Primeiro colocado R$", valor1)
print("Segundo colocado R$", valor2)
print("Terceiro colocado R$", valor3)